acquia/mc-cs-plugin-custom-objects
# MAUT-11515 : Custom object date segment filters should have have same operators than date custom field filters

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | No
| New feature/enhancement? (use the a.x branch)      | Yes
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | Yes 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #https://acquia.atlassian.net/browse/MAUT-11515 



#### Description:
Users noticed that segment filters based on custom object date/datetime fields have different operators than segment filters based on date/datetime custom objects. 


#### Steps to test this PR:


1. Open this PR for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a custom object with a date and a datetime fields 
3. Create a custom field with a date and a datetime field
4. Create a segment based on above fields and notice the operators.
5. Custom object date/time fields should have the same operators as the normal custom fields.
